apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/PublisherClient.cs

```diff
@@ -120,7 +120,19 @@ public sealed class ClientCreationSettings
                 PublisherServiceApiSettings publisherServiceApiSettings = null,
                 ChannelCredentials credentials = null,
                 string serviceEndpoint = null)
-                : this(clientCount, publisherServiceApiSettings, credentials, serviceEndpoint, EmulatorDetection.None)
+                : this(clientCount, publisherServiceApiSettings, credentials, serviceEndpoint, EmulatorDetection.None, null)
+            {
+            }
+
+            // TODO: Allow null?
```

Comment:
And if not, validate it :)

Reply:
Yep, why not, everything else may be null.